pages/deployment/aws.mdx

```diff
@@ -32,7 +32,13 @@ Memgraph supports multiple Linux distributions. Memgraph packages for supported
 from the [following page](/getting-started/install-memgraph/direct-download-links#linux)
 
 Running Memgraph natively will bring some speed improvements compared to the
-Docker version of Memgraph. 
+Docker version of Memgraph. However, deploying Memgraph with Docker is a more straightforward approach
+since it comes with built-in Memgraph MAGE algorithms.
+Memgraph MAGE contains modules that are written in C++, Python and Rust, which 
+will require manual work to enable them manually. For native deployment, check the guide on 
+[how to build Memgraph MAGE algorithms from source](/custom-query-modules#install-mage-and-import-query-modules).
+If you're trying out Memgraph for the first time, and running your own benchmarks against it, Docker is the 
+recommended way to run Memgraph, as it accelerates the time to value.
```

Comment:
```suggestion
Docker version of Memgraph. However, deploying Memgraph with Docker is a more straightforward approach
since it comes with built-in Memgraph MAGE algorithms.
Memgraph MAGE contains graph algorithms and utility modules written in C++, Python and Rust. If you decided to run Memgraph natively, then you need to build MAGE from source, which requires manual work. For native deployment, check the guide on 
[how to build Memgraph MAGE algorithms from source](/custom-query-modules#install-mage-and-import-query-modules).
If you're trying out Memgraph for the first time, and running your own benchmarks against it, Docker is the 
recommended way to run Memgraph, as it accelerates the time to value.
```